### PR TITLE
Updating to recent scijava-common that solves downstream ops bug

### PR DESCRIPTION
**Pull request by kephale**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -293,7 +294,6 @@
         <dependency>
             <groupId>org.scijava</groupId>
             <artifactId>scijava-common</artifactId>
-            <version>2.61.0</version>
```
Comment (skalarproduktraum) [inline pom.xml]: This should need a `<version>${scijava-common.version}</version>` here, not?

**Comment by kephale** [comment]
Worked without it for me actually. I ran the examples with the modified version, but happy to add.

File: pom.xml (hunk 1 above)
Comment (imagejan) [inline pom.xml]: No need to specify versions that are managed in the parent POM. You can leave away the `<version>` tag entirely.

**Comment by skalarproduktraum** [comment]
Maybe it wouldn't hurt to make it explicit :-)

File: pom.xml (hunk 1 above)
Comment (skalarproduktraum) [inline pom.xml]: Thanks @imagejan, I'll merge it as-is then 👍

**Pull request merged**